application/frontend/controllers/AuthenticationController.php

```diff
@@ -27,7 +27,10 @@ public function actionCreate(): User
         // rpOrigin is needed for WebAuthn authentication
         $rpOrigin = \Yii::$app->request->get('rpOrigin', '');
         if ($rpOrigin != '' && !in_array($rpOrigin, \Yii::$app->params['authorizedRPOrigins'])) {
-            throw new ForbiddenHttpException("Invalid rpOrigin", 1639169238);
+            $message = "Invalid rpOrigin. Received " . $rpOrigin . " authorized " .
+                var_export(\Yii::$app->params['authorizedRPOrigins'], true);
+            Yii::error($message);
```

Comment:
```suggestion
            \Yii::error($message);
```
I assume this should match the line above.

Reply:
Yep. Thanks